modules/text_server_adv/text_server_adv.cpp

```diff
@@ -6196,6 +6196,9 @@ String TextServerAdvanced::_strip_diacritics(const String &p_string) const {
 }
 
 String TextServerAdvanced::_string_to_upper(const String &p_string, const String &p_language) const {
+	if (p_string.size() == 0) {
```

Comment:
Could use `p_string.is_empty()` instead, which is the preferred way to check this (does the same internally).

Could also be relevant for `_string_to_lower`.

*Note:* Please make modifications by _amending_ the commit so that it stays as a single commit (see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request)).

Reply:
Thanks for the suggestions, changed to using is_empty() and added the same change to _string_to_lower